### ISSUE TITLE
KeyAuth Banned?

### ISSUE THREAD

**Issue by mazkdevf** [mazkdevf/kb#1]
Hello, i don't think i got hacked...

KeyAuth vanity url goes to some kid's discord server

**Comment by mazkdevf** [comment]
![image](https://user-images.githubusercontent.com/79049205/220757266-8bbb695a-4fab-425c-a4f1-1d29a640b0a2.png)

is that vanity again available?

**Comment by mazkdevf** [comment]
Hello @Its-Networking

**Comment by Its-Networking** [comment]
Hello, wym KeyAuth banned?

**Comment by mazkdevf** [comment]
Yeah

**Comment by mazkdevf** [comment]
All admin + higher got banned

**Comment by mazkdevf** [comment]
Did you also get banned from discord?

**Comment by Its-Networking** [comment]
How tf did that happen?

**Comment by mazkdevf** [comment]
I'm not sure, i was playing CS:GO with my friends and brother then I got disconnected, then I thought I got hacked or smth
Then I tried resetting password to the ACC and got text Account has been disabled, then on mail I got ban for Cybercrime or smth

**Comment by Its-Networking** [comment]
Fucking stupid as shit

**Comment by mazkdevf** [comment]
Yeah: https://discord.gg/HjfhCe4eE2 new server by mak's new account

**Comment by lIMonkieIl** [comment]
Hello my new discord is MonkieMoM

**Comment by Its-Networking** [comment]
Join the discord

**Comment by mazkdevf** [comment]
Sorry i don't have any account for discord rn

**Comment by lIMonkieIl** [comment]
I got the email like 20 mins ago I think

**Comment by mazkdevf** [comment]
I'm rn using tnaattori#1178 it is my brother's phone acc

**Comment by Its-Networking** [comment]
Make a new one?

**Comment by lIMonkieIl** [comment]
I have a uk number spare can us that ?

**Comment by lIMonkieIl** [comment]
Or we move to a new platform

**Comment by mazkdevf** [comment]
Ye got new mazkdevf#5717

**Comment by 6nz** [comment]
Marci#5592

**Comment by 6nz** [comment]
what now

**Comment by mazkdevf** [comment]
Ok add as friends

**Comment by mazkdevf** [comment]
just security check

**Comment by lIMonkieIl** [comment]
> Ye got new mazkdevf#5717

Added

**Comment by mazkdevf** [comment]
ok ill sec in bit
